src/contracts/SavingCircles.sol

```diff
@@ -24,6 +26,18 @@ contract SavingCircles is ISavingCircles, ReentrancyGuard, OwnableUpgradeable {
   mapping(address member => uint256[] ids) public memberCircles;
   mapping(address token => bool status) public allowedTokens;
 
+  /// @dev Requires circle is commissioned by checking if an owner is set
+  modifier onlyCommissioned(uint256 _id) {
+    if (_isDecommissioned(circles[_id])) revert NotCommissioned();
+    _;
+  }
+
+  /// @dev Requires address is a member by checking the mapping
+  modifier onlyMember(uint256 _id) {
```

Comment:
this would be more usable as `onlyMember(uint256 _id, address _member)` in my opinion